npm/npm_import.bzl

```diff
@@ -30,23 +30,243 @@ load("//npm/private:versions.bzl", "PNPM_VERSIONS")
 load("//npm/private:utils.bzl", _utils = "utils")
 load("//npm/private:npm_translate_lock.bzl", _npm_translate_lock_lib = "npm_translate_lock")
 
-npm_translate_lock_rule = repository_rule(
-    doc = _npm_translate_lock_lib.doc,
+_npm_translate_lock = repository_rule(
     implementation = _npm_translate_lock_lib.implementation,
     attrs = _npm_translate_lock_lib.attrs,
 )
 
 LATEST_PNPM_VERSION = PNPM_VERSIONS.keys()[-1]
 
-def npm_translate_lock(name, npm_package_lock = None, yarn_lock = None, pnpm_version = LATEST_PNPM_VERSION, **kwargs):
```

Comment:
Does this mean things like `tags` and `testonly` can no longer be specified? Is that ok or intentional?

Reply:
They never could in the first place since these are repository rules :)

Reply:
Perfect 👍